### ISSUE TITLE
Implement multidistro e2e validation for CentOS 7

### ISSUE THREAD

**Issue by msivosuo** [intel/cri-resource-manager#484]
**Describe the solution you'd like**

Implement multidistro e2e validation for CentOS 7

**Why this is needed**

To improve validation.

**Comment by klihub** [comment]
#494

**Comment by marquiz** [comment]
We have packaging tests. @askervin @klihub let's just close this?

**Comment by askervin** [comment]
@marquiz , yes, let's close. We are running (manually) e2e tests with distro=centos-7. They looked good for v0.8.0.
